NitroxLauncher/NitroxLauncher.csproj

```diff
@@ -149,6 +149,16 @@
     <Resource Include="Fonts\Roboto-Regular.ttf" />
     <Resource Include="Fonts\Roboto-Thin.ttf" />
     <Resource Include="Fonts\Roboto-ThinItalic.ttf" />
+    <Resource Include="Fonts\RobotoMono-Bold.ttf" />
+    <Resource Include="Fonts\RobotoMono-BoldItalic.ttf" />
+    <Resource Include="Fonts\RobotoMono-Italic.ttf" />
+    <Resource Include="Fonts\RobotoMono-Light.ttf" />
+    <Resource Include="Fonts\RobotoMono-LightItalic.ttf" />
+    <Resource Include="Fonts\RobotoMono-Medium.ttf" />
+    <Resource Include="Fonts\RobotoMono-MediumItalic.ttf" />
+    <Resource Include="Fonts\RobotoMono-Regular.ttf" />
+    <Resource Include="Fonts\RobotoMono-Thin.ttf" />
+    <Resource Include="Fonts\RobotoMono-ThinItalic.ttf" />
```

Comment:
Do we need to retain the license in source?

Reply:
Short answer:
I'd say yes, and it's already done.

Long answer:
Per my understanding of the Apache License v2: yes and I would say we have. I confirmed both fonts, "Roboto" and "Roboto Mono", included have the same license (the former's license was there already). I performed a binary compare between the two license files confirming they matched (summarized in 661de77 comments). IMHO a common license file in the same folder applying to both is a reasonable interpretation of the structure. We could include two identical license files with different names or org in a subfolder if you think it's necessary.

What I haven't checked is if we actually distribute the license file in the "install", which Apache requires as I understand. Also, sometimes a copyright notice is at the top of the file license file and/or included in a summary of open source technologies in the distribution...I don't see Google having listed themselves as the copyright holder anywhere of the fonts, but I probably am just not looking in the right place.

<hr>

The above is for reference only and not be construed as legal advice...

Reply:
Is this adequate and should I resolve this review?

Reply:
That sounds good to me.